packages/components/ui/src/a11y/menu/A11yMenuDriver.svelte

```diff
@@ -0,0 +1,35 @@
+<script>
+	import StorageIO from '../../io/storage/StorageIO.svelte';
+	import ColorCorrection from './ColorCorrection.svelte';
+	import {
+		_a11yColorStyles,
+		_a11yTextStyles,
+		_a11ySettings,
+		applyStyles,
+		mergeDefaultSettings,
+	} from './settings';
+
+	export let targetSelector = 'html';
+	export let defaults = null;
+	export let useLocalStorage = true;
+
+	let defaultValue;
+
+	$: defaults && (defaultValue = mergeDefaultSettings(defaults));
+	$: targetNode = globalThis.document?.querySelector(targetSelector);
```

Comment:
I'm getting `'globalThis' is not defined.` so I guess we need to `eslint-disable-next-line no-undef` here

Reply:
we could use the new `isClientSide` as well

Reply:
OK